spec/nikeplus/account_spec.rb

```diff
@@ -5,6 +5,26 @@
    NikePlus.reset_config
   end
 
+  it "delegate activities to Activities model" do
+    account = nil
+    VCR.use_cassette("succeed_token_retrieval") do
+      account = NikePlus::Account.new("foo@bar.com", "foobar")
+    end
+    VCR.use_cassette("all_activities") do
+      expect(account.activities.data.first).to be_kind_of NikePlus::Activity
+    end
+
+  end
+
+  it "cache token result" do
+      VCR.use_cassette("succeed_token_retrieval") do
+        NikePlus::Account.new("foo@bar.com", "foobar").fetch_token
+      end
+      # do not raise vcr unexpected request error message
+      NikePlus::Account.new("foo@bar.com", "foobar").fetch_token
+      expect(NikePlus.configuration).to eq({:access_token=>"thetoken"})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.